### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by toliboff**
Added media queries and stylings for the desktop version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,12 +21,25 @@
...
+                <a href="#contact">Contact</a>
+            </li>
+        </ul>
       </nav>
     </header>
 
     <section class="headline">
-      <h1>I’m Tolibjon Glad to see you!</h1>
+      <h1>I’m Tolibjon <br/>
+          Glad to see you!
```
Comment (Temmarie) [inline index.html:41]: - [ ] You must stick to the design as much as possible (e.g. font, colours, image size and placements, margins) using the templates in Figma.
- Adjust the background image in the headline to be aligned as the one in the Figma design.
_See the images below for more clarity._

**Original** | **Your clone**
  ----------- | -----------
  image | image

File: index.html [hunk 2]
```diff
@@ -59,134 +72,148 @@ <h2 class="connect-me">LET'S CONNECT</h2>
...
+                <div class="stats">
+                    <span class="company">CANOPY</span>
+                    <span class="role-and-year">Back End Dev</span>
+                    <span class="role-and-year">2015</span>
+                </div>
 
-          <p class="description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
```
Comment (Temmarie) [inline index.html:77]: - [ ] Kindly adjust the image and project-info to fit in the required change. 
- The image should occupy half of the card, increase the height and width.
- The project-info should be properly aligned horizontally. The `role-and-year` style is also affecting the required design for that section.
_**Refer to my comments above about the design.**_ 
_See the images below for clarity._

**Original** | **Your clone**
  ----------- | -----------
  image | image

File: index.html [hunk 3]
```diff
@@ -59,134 +72,148 @@ <h2 class="connect-me">LET'S CONNECT</h2>
...
-            <li class="language">html</li>
-            <li class="language">css</li>
-            <li class="language">javascript</li>
-          </ul>
+                <ul class="languages">
+                    <li class="language">html</li>
+                    <li class="language">Ruby on Rails</li>
+                    <li class="language">css</li>
+                    <li class="language">javascript</li>
+                </ul>
```
Comment (Temmarie) [inline index.html:122]: - [ ] Kindly adjust the image and project-info to fit in the required change. 
- The image should occupy half of the card, increase the height and width.
- The project-info should be properly aligned horizontally. The `role-and-year` style is also affecting the required design for that section.
- Make the same changes for the remaining projects.
_**Refer to my comments above about the design.**_ 
_See the images below for clarity._

**Original** | **Your clone**
  ----------- | -----------
  image | image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @toliboff and @Metsanpeitto     ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 

Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. :smile:
- - -

### **Required Changes** ♻️
- [ ] Kindly refer back to the Figma design and adjust your styles to be as similar as possible to the template.
_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @toliboff @Metsanpeitto🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

doing good

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**